pkg/declarativeconfig/permission_set.go

```diff
@@ -0,0 +1,33 @@
+package declarativeconfig
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"gopkg.in/yaml.v3"
+)
+
+type PermissionSet struct {
+	Name             string              `yaml:"name,omitempty"`
+	Description      string              ` yaml:"description,omitempty"`
+	ResourceToAccess map[string]DCAccess `yaml:"resource_to_access,omitempty"`
```

Comment:
I don't think we should start using `_` in our YAMLs.

I think we can re-use some of the principles / guidelines we have for [operator CR and their naming conventions](https://github.com/stackrox/stackrox/blob/master/operator/EXTENDING_CRDS.md#naming), and would propose to use camel case instead (It also feels awkward writing a YAML with underscores, but that's more of a personal opinion :))

Additionally, we are not bound to re-use proto names or others within our YAMLs, so maybe we want to switch away from `resourceToAccess`, to something more simple, such as `resources`?